modules/core/perf/perf_convertTo.cpp

```diff
@@ -4,7 +4,7 @@ namespace opencv_test
 {
 using namespace perf;
 
-typedef tuple<Size, MatType, MatType, int, double> Size_DepthSrc_DepthDst_Channels_alpha_t;
+typedef tuple<Size, MatDepth, MatDepth, int, double> Size_DepthSrc_DepthDst_Channels_alpha_t;
```

Comment:
I started receiving the error `No regression data for dst argument` as in [this build](http://pullrequest.opencv.org/buildbot/builders/precommit_linux64/builds/17584).
@alalek Any idea how to resolve that?

Reply:
Avoid changing test/parameter names of performance tests.

```.diff
-Size_DepthSrc_DepthDst_Channels_alpha_convertTo--convertTo---640x480--8UC1--8UC1--1--1-
+Size_DepthSrc_DepthDst_Channels_alpha_convertTo--convertTo---640x480--CV_8U--CV_8U--1--1-
```

Reply:
I can delay that for the time being, but this needs to be resolved.
The problem is that the parameters are actually MatDepth not MatType, and in case of enum types, an explicit cast is needed.

Are regression files available for public? Can we add some alias nodes?

Reply:
XML files are here: https://github.com/opencv/opencv_extra/tree/master/testdata/perf

Run perf tests (Debug build / without IPP, any optimizations) with `--perf_write_sanity --perf_min_samples=1 --perf_force_samples=1` to generate new.